### PR TITLE
CB-194: minors 5

### PR DESCRIPTION
**Pull request by diegoAndres01**
[Card Jira](https://inube.atlassian.net/browse/CB-194)

### CODE REVIEW

**Review by novadeveloper14 — APPROVED** [review]

File: src/pages/board/outlets/creditProfileInfo/index.tsx [hunk 1]
```diff
@@ -177,73 +201,100 @@ export const CreditProfileInfo = () => {
...
-            />
+            </>
           )}
+          <Button
+            onClick={handlePrint}
+            disabled={isGeneratingPdf}
+            spacing="compact"
+            variant="filled"
+          >
+            Imprimir
```
Comment (carin91) [inline src/pages/board/outlets/creditProfileInfo/index.tsx:256]: Relocated that into a config file

File: src/pages/board/outlets/creditProfileInfo/index.tsx [hunk 2]
```diff
@@ -177,73 +201,100 @@ export const CreditProfileInfo = () => {
...
-              <Text
-                type="headline"
-                size={isMobile ? "small" : "medium"}
-                textAlign="center"
-              >
-                {requests.nnasocia
-                  ? capitalizeFirstLetterEachWord(requests.nnasocia)
-                  : ""}
+              <Text type="title" size="medium" appearance="gray" weight="bold">
+                Perfil crediticio
```
Comment (carin91) [inline src/pages/board/outlets/creditProfileInfo/index.tsx:270]: Relocate every spanish word in a config file

**Review by carin91 — APPROVED** [review]

File: src/pages/board/outlets/creditProfileInfo/index.tsx (hunk 1 above)
Comment (diegoAndres01) [reply src/pages/board/outlets/creditProfileInfo/index.tsx]: resolved

File: src/pages/board/outlets/creditProfileInfo/index.tsx (hunk 2 above)
Comment (diegoAndres01) [reply src/pages/board/outlets/creditProfileInfo/index.tsx]: resolved

**Pull request merged**